ta4j-core/src/main/java/org/ta4j/sparx/criteria/CompoundedAnnualGrowthReturnCriteria.java

```diff
@@ -0,0 +1,41 @@
+package org.ta4j.sparx.criteria;
+
+import org.ta4j.core.BarSeries;
+import org.ta4j.core.Position;
+import org.ta4j.core.TradingRecord;
+import org.ta4j.core.criteria.AbstractAnalysisCriterion;
+import org.ta4j.core.criteria.pnl.ReturnCriterion;
+import org.ta4j.core.num.DoubleNum;
+import org.ta4j.core.num.Num;
+
+public class CompoundedAnnualGrowthReturnCriteria extends AbstractAnalysisCriterion {
+    @Override
```

Comment:
Have you run the auto formatter (i.e. `mvn -B clean license:format formatter:format test`)? If not you'll need to do so as it's a pre-req for merging.

Reply:
hi @TheCookieLab . Apologies, I've mistakenly raise this PR. I think I have closed this PR but pleast let me know if it is still showing up. I was supposed to just raise a PR to my forked repo.
But thank you for reviewing the PR learned a lot from them.